bin/build-hdfs

```diff
@@ -9,6 +9,7 @@ IMPALA_ZIP=$IMPALA_VER.zip
 IMPALA_URL=https://github.com/cloudera/Impala/archive/$IMPALA_ZIP
 NATIVE=native
 DIST=hdfs-mesos-0.1.0
+DOCKER_IMAGE_NAME=hdfs-mesos
```

Comment:
Change this to be `${DOCKER_IMAGE_NAME:-"hdfs-mesos"} so that the name can be overridden from outside the script if necessary.